api/cmd/managerapi.go

```diff
@@ -62,7 +63,7 @@ func NewManagerAPICommand() *cobra.Command {
 				var newMws []droplet.Middleware
 				// default middleware order: resp_reshape, auto_input, traffic_log
 				// We should put err_transform at second to catch all error
-				newMws = append(newMws, mws[0], &handler.ErrorTransformMiddleware{})
+				newMws = append(newMws, mws[0], &handler.ErrorTransformMiddleware{}, &filter.AuthenticationMiddleware{})
```

Comment:
just confirm that will other filters be executed if user is not login?

Reply:
Yes. It will.
Those middlewares are in the gin's API handler.